config/v1/types_network.go

```diff
@@ -13,20 +13,50 @@ type Network struct {
 	// Standard object's metadata.
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// spec holds user settable values for configuration
+	// Spec holds user settable values for configuration.
 	Spec NetworkSpec `json:"spec"`
-	// status holds observed values from the cluster. They may not be overridden.
+	// Status holds observed values from the cluster. They may not be overridden.
 	Status NetworkStatus `json:"status"`
 }
 
+// NetworkSpec is the desired network configuration.
+// As a general rule, this SHOULD NOT be used directly. Instead, you should
+// consume the NetworkStatus, as it indicates the currently deployed configuration.
 type NetworkSpec struct {
-	// serviceCIDR
-	// servicePortRange
-	// vxlanPort
-	// 	ClusterNetworks    []ClusterNetworkEntry `json:"clusterNetworks"`
+	// IP address pool to use for pod IPs.
+	ClusterNetworks []ClusterNetwork `json:"clusterNetworks"`
+
+	// IP address pool for services.
+	// Currently, we only support a single entry here.
+	ServiceNetworks []string `json:"serviceNetworks"`
+
+	// NetworkType is the plugin that is to be deployed (e.g. OpenShiftSDN).
+	// N.B. changing this is currently not supported.
+	NetworkType string `json:"networkType"`
```

Comment:
You need to explain what the set of possible values is or at least indicate where the canonical list is

Reply:
> changing this is currently not supported.

nit: specify that the change is not supported after installation.

Reply:
Why not just making this a type and create constants matching the possible values. This will limit the scope of errors. We this in several places already.